fastlane/lib/fastlane/actions/pod_lib_lint.rb

```diff
@@ -27,6 +27,7 @@ def self.run(params)
         command << "--skip-import-validation" if params[:skip_import_validation]
         command << "--skip-tests" if params[:skip_tests]
         command << "--analyze" if params[:analyze]
+        command << "--configuration=#{params[:configuration]}" if params[:configuration]
```

Comment:
What happens if a configuration has a space in the name? 🤔 

I'm thinking maybe we need to either wrap this in quotes or call `.shellescape` on it? Thoughts?